modules/calendar.md

```diff
@@ -63,7 +63,7 @@ The following properties can be configured:
+| `customEvents`                        | An array of `keyword`/`symbol`/`color`/`transform` that will customize events based on keyword in title. <br> <br> `keyword` is a case-insensitive string that if present in event title will trigger the use of custom symbol and/or color for that event,<br>`symbol` is the Font Awesome icon to use as symbol and<br> `color` is the CSS color to use for the event.<br>`transform` is an object for regular expression text replacement of the calendar title consisting of the following elements:<br>- `search` regular expression to search for<br>- `replace` replace string, may contain group match references<br>- `yearmatchgroup` (optional) the regex match group of a year that you want to replace to the current age<br><br>`keyword` and at least one of `symbol`, `color`, `transform` is required. <br> <br> **Examples:** <br>`customEvents: [{keyword: 'Birthday', symbol: 'birthday-cake', color: 'Gold'}]`<br><br>`customEvents: [{keyword: 'Geburtstag', symbol: 'birthday-cake', color: 'Gold', transform: { search: '^([^\']*) \'(\\d{4})$' , replace: '$1 ($2.)', yearmatchgroup: 2}},{keyword: 'in Hamburg', transform: { search: ' in Hamburg$' , replace: ''}} ]`<br>Will transform<br>![before](https://user-images.githubusercontent.com/54073894/278068297-67ca65f4-24c3-46a8-bee8-0519e4bba3f5.png)<br>to<br>![after](https://user-images.githubusercontent.com/54073894/278068458-0b4af07d-d3d9-4644-a4a6-e8c402598208.png)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
just one more thing: put images into the `modules/screenshot` folder

Reply:
done :)